pkgs/by-name/ka/kanban-bash/package.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  gawk,
+  which,
+  gnused,
+  gnugrep,
+  ncurses,
+  coreutils,
+  unixtools,
+  stdenvNoCC,
+  makeWrapper,
+  fetchpatch2,
+  fetchFromGitHub,
+  installShellFiles,
+  writableTmpDirAsHomeHook,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "kanban.bash";
+  version = "0-unstable-2024-10-17";
+  src = fetchFromGitHub {
+    owner = "coderofsalvation";
+    repo = "kanban.bash";
+    rev = "325e2c0c3e75f6bb2106d15107559bc1eea6fd5d";
+    hash = "sha256-OE4kKixNG172yMR5vp5O4STq9fGdJVWLIt7j6deNxhI=";
+  };
+  patches = [
+    # bug fix for linux
+    (fetchpatch2 {
+      url = "https://github.com/coderofsalvation/kanban.bash/pull/45.patch?full_index=1";
```

Comment:
maybe this could be merged?

Reply:
We tried that but upstream seems to be not that responsive. 😅